packages/celotool/src/lib/geth.ts

```diff
@@ -189,7 +186,7 @@ export const checkGethStarted = (dataDir: string) => {
 }
 
 export const getWeb3AndTokensContracts = async () => {
-  const kit = newKit('http://localhost:8545')
+  const kit = newKitFromWeb3(new Web3('http://localhost:8545'))
```

Comment:
The problem that I see with adding the web3 to the constructor, is that we are trying to abstract that with the communication/connect layer.
The natural way, using constructors for this will be:
```
const web3 = new Web3(url)
const connect = new Connect(web3)
const kit = new ContractKit(connect)
```
that why the factory way of reducing that